insights/config/specs.py

```diff
@@ -414,7 +414,7 @@
     "docker_images"             : "/usr/bin/docker images -q",
     "dumpdev"                   : "/bin/awk '/ext[234]/ { print $1; }' /proc/mounts",
     "getblockschedulers"        : "for device in $(ls /sys/block); do echo /sys/block/$device/queue/scheduler; done",
-    "httpd_pid"                 : "ps aux | grep httpd | grep -v grep | head -1 | awk '{print $2}'",
+    "httpd_pid"                 : "ps aux | grep /usr/sbin/httpd | grep -v grep | head -1 | awk '{print $2}'",
```

Comment:
I was going to suggest that this wasn't as accurate as the suggestion from https://access.redhat.com/solutions/1346533 to use:

```bash
systemctl show httpd -p MainPID | cut -d = -f 2
```
But that won't work on RHEL 6 systems.  I'm not keen on running long pipelines of shell commands to get data, but at the moment we don't seem to have a good way to get this kind of information in any other way, so I think that's OK.

Reply:
Just as you said, the way I used, is to make it compatible for both RHEL6 and RHEL7, Thanks @PaulWay